pkg/microservice/aslan/core/common/repository/mongodb/template/product.go

```diff
@@ -82,8 +83,8 @@ func (c *ProductColl) FindProjectName(project string) (*template.Product, error)
 }
 
 func (c *ProductColl) ListNames(inNames []string) ([]string, error) {
-	res, err := c.listProjects(inNames, bson.D{
-		{"product_name", 1},
+	res, err := c.listProjects(inNames, bson.M{
+		"product_name": "$product_name",
```

Comment:
what the reason for it?

Reply:
use Projection to query necessary fields